examples/oas/forged_oas3.json

```diff
@@ -0,0 +1,320 @@
+{
```

Comment:
What does forged_oas3 mean?

Reply:
It is a faulty oas3 doc. Its created by modifying the actual/genuine oas doc of petstore.
It is used for testing purpose like checking the behavior of our app against testcases generated from a faulty oas3 doc.